templates/base_index.html

```diff
@@ -80,7 +80,7 @@
     <link rel="stylesheet" type="text/css" media="screen" href="{{ versioned_static('css/styles.css') }}" />
   </head>
 
-  <body {% if is_paper %} class="is-paper" {% endif %}>
+  <body class="is-paper">
```

Comment:
Are all pages ready for this?

Reply:
Thanks for reviewing this, I'm on it. For the paper colour - the entire site is being deployed at once on 4th July, so yes - we can safely get rid of that logic and just use `is-paper` everywhere.

P.S. There are 6 more prs ready for review if you have a moment :)